### PR TITLE
chore: Improves config options description, adds defaults and uses string types

### PR DESCRIPTION
**Pull request by saltiyazan**
# Description

- Improves config options description and adds defaults.
- Uses strings as required by the exec command (solves #66 )

# Checklist:

- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [ ] I validated that new and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have bumped the version of the library

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -10,20 +10,23 @@ options:
     type: string
     description: URL of the custom API that can be used to fulfill the DNS-01 challenge
   httpreq_http_timeout:
-    type: int
-    description: API request timeout
+    type: string
```
Comment (Kayra1) [inline config.yaml:13]: Does this type matter? I don't see any type conversions in the code.

**Review by Kayra1 — APPROVED** [review]

File: config.yaml (hunk 1 above)
Comment (saltiyazan) [reply config.yaml:13]: Yes, to my understanding the error in #66 states that when executing a command in the workload the environment variables need to be strings. We use strings in other lego charms.

**Comment by saltiyazan** [comment]
Is there a reason not to promote the charm to stable after merging this PR? @gruyaume 
The bug fixed by this PR is on stable too.

**Review by gruyaume — APPROVED** [review]

**Comment by gruyaume** [comment]
> Is there a reason not to promote the charm to stable after merging this PR? or should we go ahead and do so? @gruyaume The bug fixed by this PR is on stable too.

I'm ok with promoting it to stable @saltiyazan

**Pull request merged**